src/Analyzers/CSharp/Tests/UseInterpolatedString/CSharpUseInterpolatedStringTests.cs

```diff
@@ -0,0 +1,245 @@
+        }
+        """,
+        FixedCode = """
+        class Program
+        {
+            private string MyString1 => $"MyString";
+            private string MyString2 => $"My{{String";
+            private string MyString3 => $"My}}String";
+            private string MyString4 => $"My{{S}}tring";
+            private string MyString5 => $"My}}S{{tring";
+        }
+        """,
+        ExpectedDiagnostics =
+        {
+            // /0/Test0.cs(3,33): hidden IDE0330: String can be converted to interpolated string
+            VerifyCS.Diagnostic().WithSpan(3, 33, 3, 43),
+            // /0/Test0.cs(4,33): hidden IDE0330: String can be converted to interpolated string
+            VerifyCS.Diagnostic().WithSpan(4, 33, 4, 44),
+            // /0/Test0.cs(5,33): hidden IDE0330: String can be converted to interpolated string
+            VerifyCS.Diagnostic().WithSpan(5, 33, 5, 44),
+            // /0/Test0.cs(6,33): hidden IDE0330: String can be converted to interpolated string
+            VerifyCS.Diagnostic().WithSpan(6, 33, 6, 45),
+            // /0/Test0.cs(7,33): hidden IDE0330: String can be converted to interpolated string
+            VerifyCS.Diagnostic().WithSpan(7, 33, 7, 45),
+        }
+    }.RunAsync();
+
+    [Fact]
+    public Task FixVerbatimStrings() => new VerifyCS.Test
+    {
+        TestCode = """
+        class Program
+        {
+            private string MyString1 => @"MyString";
+            private string MyString2 => @"My{String";
+            private string MyString3 => @"My}String";
+            private string MyString4 => @"My{S}tring";
+            private string MyString5 => @"My}S{tring";
+        }
+        """,
+        FixedCode = """
+        class Program
+        {
+            private string MyString1 => $@"MyString";
+            private string MyString2 => $@"My{{String";
+            private string MyString3 => $@"My}}String";
+            private string MyString4 => $@"My{{S}}tring";
+            private string MyString5 => $@"My}}S{{tring";
+        }
+        """,
+        ExpectedDiagnostics =
+        {
+            // /0/Test0.cs(3,33): hidden IDE0330: String can be converted to interpolated string
+            VerifyCS.Diagnostic().WithSpan(3, 33, 3, 44),
+            // /0/Test0.cs(4,33): hidden IDE0330: String can be converted to interpolated string
+            VerifyCS.Diagnostic().WithSpan(4, 33, 4, 45),
+            // /0/Test0.cs(5,33): hidden IDE0330: String can be converted to interpolated string
+            VerifyCS.Diagnostic().WithSpan(5, 33, 5, 45),
+            // /0/Test0.cs(6,33): hidden IDE0330: String can be converted to interpolated string
+            VerifyCS.Diagnostic().WithSpan(6, 33, 6, 46),
+            // /0/Test0.cs(7,33): hidden IDE0330: String can be converted to interpolated string
+            VerifyCS.Diagnostic().WithSpan(7, 33, 7, 46),
+        }
+    }.RunAsync();
+
+    [Fact]
+    public Task FixSingleLineRawStrings() => new VerifyCS.Test
```

Comment:
I got some tests to pass, but this test and the next one are failing for a reason I cannot understand.
![image](https://github.com/dotnet/roslyn/assets/7676354/5d756d2b-4b45-4943-a05d-0285781beb9e)
Even though I'm using the right start/end tokens in https://github.com/dotnet/roslyn/pull/73008/files#diff-db1e05fa98ce7807a912e3f653c71fb03ce8fb5eb7e96d0e39b585f0858cceeaR66-R87
I'm not seeing that in the `FixedCode`. Any idea why this would be?

Reply:
i see the issue.  i'll comemnt accordingly.